test/OpenTelemetry.Exporter.Zipkin.Tests/OpenTelemetry.Exporter.Zipkin.Tests.csproj

```diff
@@ -5,6 +5,9 @@
     <TargetFrameworks Condition="$(OS) == 'Windows_NT'">$(TargetFrameworks);net461</TargetFrameworks>
     <IsPackable>false</IsPackable>
   </PropertyGroup>
+  <ItemGroup>
+    <Compile Include="..\OpenTelemetry.Collector.Dependencies.Tests\TestServer.cs" Link="TestServer.cs" />
```

Comment:
perhaps move this file in a separate folder.

Reply:
Maybe I should move TestServer to OpenTelemetry.Tests project and then link back into Dependencies.Tests & Zipkin.Tests? Only because it seems likely to be shared by more in the future.